webview_panels/src/modules/defer/DeferToProduction.tsx

```diff
@@ -93,27 +111,80 @@ const DeferToProduction = (): JSX.Element => {
     }
   };
 
-  const handleManifestPathChange = (
+  const handleLocalManifestPathChange = (
     event: React.ChangeEvent<HTMLInputElement>,
   ) => {
     const { value, name } = event.target;
     setDeferState((prevState) => ({ ...prevState, [name]: value }));
   };
 
-  const onManifestBlur = async () => {
+  const onLocalManifestBlur = async () => {
     const response = await executeRequestInSync("updateDeferConfig", {
-      key: "manifestPathForDeferral",
-      value: manifestPathForDeferral,
+      key: "localManifestPathForDeferral",
+      value: localManifestPathForDeferral,
       isPreviewFeature: true,
     });
     if (!(response as { updated: boolean }).updated) {
       setDeferState((prevState) => ({
         ...prevState,
-        manifestPathForDeferral: "",
+        localManifestPathForDeferral: "",
       }));
     }
   };
 
+  const handleManifestPathTypeChange = async (option: string) => {
+    if (option === "remote") {
+      const response = await executeRequestInSync(
+        "fetchProjectIntegrations",
+        {},
+      );
+      if (Array.isArray(response)) {
+        setDeferState((prevState) => ({
+          ...prevState,
+          projectIntegrations: response.map(
+            (item: { name: string; id: number }) => ({
+              label: item.name,
+              value: item.id,
+            }),
+          ),
+          manifestPathType: option,
+        }));
+        return;
+      }
+    }
+    setDeferState((prevState) => ({
+      ...prevState,
+      manifestPathType: option,
+    }));
+  };
+
+  const handleIntegrationSelect = async (selectedOption: {
+    label: string;
+    value: number;
+  }) => {
+    const response = await executeRequestInSync("downloadManifest", {
```

Comment:
The manifest should be downloaded prior to dbt command execution. We should guarantee that the latest artifact is used for any command execution.

Reply:
Yes. As soon as the user selects any of the project integrations, it will automatically download the latest artifact.